components/prize_pool/commons/prize_pool.lua

```diff
@@ -911,6 +914,10 @@ function Placement:_getLpdbData()
 	return entries
 end
 
+function Placement:_getPrizeRewardForOpponent(opponent, prize)
```

Comment:
imo export it, so it is also accessible in /Custom for e.g. secondary points storage
alternatively do the secondary points storage also via commons

Reply:
as requested